addons/pos_restaurant/static/src/app/services/pos_store.js

```diff
@@ -882,4 +884,35 @@ patch(PosStore.prototype, {
         order.selectCourse(destCourse);
         order.recomputeOrderData();
     },
+    async loadSampleData() {
+        if (this.config.module_pos_restaurant) {
+            const data = { screen: "ProductScreen" };
+            const table_number = this.getOrder()?.table_id?.table_number;
+            if (table_number) {
+                data.tableNumber = table_number;
+            }
+            sessionStorage.setItem("posPreSampleDataLoadState", JSON.stringify(data));
+        }
+        return super.loadSampleData();
+    },
+    restoreSampleDataSate() {
```

Comment:
little typo in the method name I think
```suggestion
    restoreSampleDataState() {
```